### DISCUSSION TITLE
Local dev "npm start" does not work

### DISCUSSION THREAD

**Discussion by ClarkDuck** [mozilla/hubs#4856] [Hubs Cloud Questions]
**Description**
App Home page don't work.
Firstly it was appear for 0.1 sec only once by F5 refresh page, but it is disappear and screen is white blank.

**To Reproduce**
`git checkout hubs-cloud`
`npm ci`
`npm run login`
`npm start`
goto https://localhost:8080/webpack-dev-server/

**Expected behavior**
landing page should be appear.

**Screenshots**
![Screenshot from 2021-10-18 08-29-33](https://user-images.githubusercontent.com/92729925/137730940-65b014b7-e076-48cb-9328-f59fe4fcabb4.png)

**Hardware** 
 - Device: Desktop
 - OS: Ubuntu 18
 - Browser: Chrome 94.0.4606.61 (Official Build) (64-bit)

**Additional context**
backend version: Hubs Cloud Personal 1.1.2

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/HUBS-1108)

**Comment by camelman90** [comment]
These white pages often indicate that there is an issue with your LocalStorage data.
Can you try removing the "__hubs_store" Local Storage entry and check?

**Comment by ClarkDuck** [comment]
Thanks,
It works for only landing page, but I after signin process, if I goto the next page by clicking the "CreateRoom" button, then it show again white blank.

Here is screenshot for login.
![Screenshot from 2021-10-19 04-21-51](https://user-images.githubusercontent.com/92729925/137871597-2b7debed-88ed-489e-9c5a-c72db9324d3a.png)

**Reply by nupur711** [reply]
> Thanks, It works for only landing page, but I after signin process, if I goto the next page by clicking the "CreateRoom" button, then it show again white blank.
> 
> Here is screenshot for login. ![Screenshot from 2021-10-19 04-21-51](https://user-images.githubusercontent.com/92729925/137871597-2b7debed-88ed-489e-9c5a-c72db9324d3a.png)

same issue with me. did you find the solution?

**Comment by camelman90** [comment]
I never used the webpack dev server output, but directly go to hub.html. When I develop custom clients, I use URL like this:

`https://localhost:8080/hub.html?hub_id=yMhWF8C`

Looks like webpack dev server is trying to embed the page, which does not work because Hubs uses a specific configuration when embedding and therefore you are missing an embed token in the parameters (&embed_token="something"). Try accessing it directly like the link above instead of the webpack server. A new issue could be made for "no feedback when missing embed token", it should be communicated more clearly in the interface I guess.

Worst case you can find embed token from console: window.APP.hub.embed_token, but I dont think this is the proper way to set up a development environment.

You may also find more resources in the hubs developer discord https://discord.gg/vYFEt9v4VH

**Comment by ghost** [comment]
Having the same issue did you find the solution?
